### PR TITLE
[docs] Add unleash feature flags guide.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-20665

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "grammarly.selectors": [
+    {
+      "language": "markdown",
+      "scheme": "file"
+    }
+  ]
+}
```
Comment (arivepr) [inline .vscode/settings.json:8]: Linters complaining about no new line at eof here 🤮

**Review by arivepr — APPROVED** [review]
Docs look great!

File: .vscode/settings.json (hunk 1 above)
Comment (Hyperkid123) [reply .vscode/settings.json:8]: Why is this file even included?

Comment (Hyperkid123) [reply .vscode/settings.json]: Its gone

Comment (arivepr) [reply .vscode/settings.json]: yoiu're starting to sound like me 🤣

**Pull request merged**